backend/post-service/src/main/resources/requests/PostController/post-GET.http

```diff
@@ -0,0 +1,6 @@
+### GET request from owner of the profile
+GET http://localhost:8082/api/posts/
+    6c84fb95-12c4-11ec-82a8-0242ac130003?
```

Comment:
Taka sugestia, ze jak testujemy endpoint z zewnątrz to może używać portu Gateway'a, żeby poprawnie za każdym razem testować działanie całego systemu.